protocol_driver_mercury.cc

```diff
@@ -36,7 +36,7 @@ ProtocolDriverMercury::ProtocolDriverMercury() {}
 absl::Status ProtocolDriverMercury::Initialize(
     const ProtocolDriverOptions& pd_opts, int* port) {
   std::string netdev_name = pd_opts.netdev_name();
-  auto maybe_ip = IpAddressForDevice(netdev_name, 4);
+  auto maybe_ip = IpAddressForDevice(netdev_name, mercury_ipv4_only_ ? 4 : 0);
```

Comment:
Not sure I understand this. What is `mercury_ipv4_only_` ?

Reply:
your explanation below was correct.